test/core/TakerOperations.t.sol

```diff
@@ -641,37 +593,15 @@ contract TakerOperationsTest is MangroveTest {
     quote.approve($(mgv), 1 ether);
     expectFrom($(mgv));
     emit OfferFail($(base), $(quote), ofr, $(this), 1 ether, 1 ether, "mgv/makerRevert");
-    testMgv.snipesInTest($(base), $(quote), wrap_dynamic([ofr, uint(Tick.unwrap(offerTick)), 1 ether, 50_000]), true);
-  }
-
-  function test_snipe_on_higher_price_fails() public {
-    uint ofr = mkr.newOfferByVolume(1 ether, 1 ether, 100_000, 0);
-    Tick offerTick = pair.offers(ofr).tick();
-    quote.approve($(mgv), 0.5 ether);
-
-    (uint successes,,,,) = testMgv.snipesInTest(
-      $(base), $(quote), wrap_dynamic([ofr, uint(Tick.unwrap(offerTick) - 1), 1 ether, 100_000]), true
-    );
-    assertTrue(successes == 0, "Order should fail when order price is higher than offer");
-  }
-
-  function test_snipe_on_higher_gas_fails() public {
```

Comment:
Do we have tests for clean on that topic?

Reply:
Yes, `test_giving_smaller_gasreq_to_clean_fails` in the same file.

Reply:
Sorry I should have checked better that all those tests I asked about had cleaning counterparts!

Reply:
np :-)